client/src/controllers/AutoSizeController.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller } from '@hotwired/stimulus';
+import autosize from 'autosize';
+
+export class AutoSizeController extends Controller {
+  connect() {
+    autosize(this.element);
+    this.element.addEventListener('input', () => {
+      autosize.update(this.element);
+    });
```

Comment:
What's the need for this? my understanding from the docs is that we would not need this listener, it's automatically handled by the library.

http://www.jacklmoore.com/autosize/